### ISSUE TITLE
Check if order is completed.

### ISSUE THREAD

**Issue by JakobWestergarden** [fhqvst/avanza#65]
After reading through the API documentation and doing some testing on my own, i still don't see if there is a function to check if one of your orders is still on the market or has been sold/bought.  Right now i am using editOrder and check if i get an error message with info that order is no longer on market, but is there a better way to do this without having to edit the order and just getting the info on whether the order is completed or not?

**Comment by fhqvst** [comment]
You should be able to subscribe to `Avanza.ORDERS`, like so:
```jsx
avanza.subscribe(Avanza.ORDERS, (orders) => {
  console.log('Received order update:', orders)
})
```

**Comment by JakobWestergarden** [comment]
> You should be able to subscribe to `Avanza.ORDERS`, like so:
> 
> ```js
> avanza.subscribe(Avanza.ORDERS, (orders) => {
>   console.log('Received order update:', orders)
> })
> ```

Thanks, that's exactly what i was looking for!

**Comment by JakobWestergarden** [comment]
When trying to use the aforementioned function, i am uncertain of how to properly call it with the correct arguments. I have tried with multiple formats, including one string with the accountId i want to subscribe on, 2 ids as one string "123456, 122456", aswell as 2 ids in an array ["123456", "122456"], but none of them seems to work for me. I also tried using your function straight up without an argument for id, but it did not work either. Am i misinterpreting something or what is my issue?

Thanks in advance
